_data/projects/learn-js.yml

```diff
@@ -0,0 +1,12 @@
+name: LearnJS
+desc: A website having collection of javascript programs and questions. 
+site: https://github.com/akulsr0/learn-js
+tags:
+  - javascript
+  - programs
+  - questions
+  - algorithm
+  - data-structures
+upforgrabs:
+  name: up for grabs
```

Comment:
This label hasn't been created in the project - should this instead be `help wanted`?